site/_includes/layouts/base.njk

```diff
@@ -33,10 +30,10 @@
     {% endif %}
 
     {# This needs to be include'd, because it expands 11ty variables #}
-    <script>{% include 'partials/script.js' %}</script>
```

Comment:
script.js needs to be removed.

it included some logic regarding cookies though, so we need to make sure this is reimplemented somewhere.